tasks/DroneFlightControlTask.hpp

```diff
@@ -111,6 +111,7 @@ namespace drone_dji_sdk
       private:
         int64_t mFunctionTimeout;
         int mStatusFreqInHz;
+        base::Time mPrevDataTimestamp;
         double mPositionThreshold;
```

Comment:
not related to this specific change, but aren't we missing a default value definition for the primitives types, or you guys are initializing them after the construction? Otherwise, using them without previously initializing wouldn't lead to an UB?

Reply:
`mFunctionTimeout`, `mStatusFreqInHz` and `mPositionThreshold` are initialized on the configureHook (so there are not used without initialization). `mPrevDataTimestamp` is initialized to Time(0) by default.

Reply:
Even though, it would be best to initialize them here as well (just removes possibility of errors). No need to change it right now, just something to remind ourselves about.

I updated the style guide to add something about this.

Reply:
I initialized the simpler ones, I'll get back to the other once we have more time.